index.html

```diff
@@ -38,5 +41,133 @@ <h1 class="name-tag">
+          </div>
+        </div>
+      </div>
+
+      <div class="about-container">
+        <img class="work1" src="works/work3.svg" alt="first project" />
+        <div class="background-card">
+          <p class="project-name">Project name goes here</p>
+
+          <ul>
+            <li class="blue-bg">HTML/CSS</li>
+            <li class="blue-bg">Ruby on Rails</li>
+            <li class="blue-bg">Javascript</li>
+          </ul>
+          <div class="arrow-link">
+            <a class="link" href="#">See this Project</a>
+            <img class="arrow-icon" src="icons/Union.svg" alt="arrow" />
+          </div>
+        </div>
+      </div>
+
+      <div class="about-container">
+        <img class="work1" src="works/work4.svg" alt="first project" />
+
+        <div class="background-card">
+          <p class="project-name">Project name goes here</p>
+
+          <ul>
+            <li class="blue-bg">HTML/CSS</li>
+            <li class="blue-bg">Ruby on Rails</li>
+            <li class="blue-bg">Javascript</li>
+          </ul>
+          <div class="arrow-link">
+            <a class="link" href="#">See this Project</a>
+            <img class="arrow-icon" src="icons/Union.svg" alt="arrow" />
+          </div>
+        </div>
+      </div>
+
+      <div class="about-container">
+        <img class="work1" src="works/work5.svg" alt="first project" />
+
+        <div class="background-card">
+          <p class="project-name">Project name goes here</p>
+
+          <ul>
+            <li class="blue-bg">HTML/CSS</li>
+            <li class="blue-bg">Ruby on Rails</li>
+            <li class="blue-bg">Javascript</li>
+          </ul>
+          <div class="arrow-link">
+            <a class="link" href="#">See this Project</a>
+            <img class="arrow-icon" src="icons/Union.svg" alt="arrow" />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="about-section">
+        <h3>About</h3>
+        <p class="about-section-tag">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ea eiusmod
+        </p>
+        <button class="dark-bg-cv" type="button">
+          Get my CV<img
+            class="down-icon"
+            src="icons/ic_down.svg"
+            alt="arrow-down"
+          />
+        </button>
+      </div>
+    </section>
+
+    <div>
+      <div class="mask-container">
+        <div>
+          <img class="mask-group" src="Images/MaskGroup1.svg" alt="languages" />
+        </div>
+        <div>
+          <img
+            class="mask-group"
+            src="Images/MaskGroup3.svg"
+            alt="frameworks"
+          />
+        </div>
+        <div class="skills-card">
+          <img class="mask-group" src="Images/MaskGroup4.svg" alt="skills" />
+        </div>
```

Comment:
- [ ] Kindly don't use images as content, instead of using images you can just hard code the elements in the `ul` tag and wrap each element in the `li` tag to make your project more professional. you need to make a list of the  languages, and technologies using HTML elements instead of making them as one image containing the snapshot of them